aiida/plugins/entry.py

```diff
@@ -80,21 +80,21 @@ def entry_points(self):
         if not self._entry_points:
             for category in self.entry_point_categories:
                 key = 'aiida.' + category
-                ep_list = [i[:i.find('=')].strip() for i in self.entry_points_raw[key]]
+                ep_list = [i[:i.find()].strip() for i in self.entry_points_raw[key]]
```

Comment:
Did you remove these `'='` from the `find()` call intentionally?

Reply:
Whoops, no, that was a search-replace mistake.  I found a couple of others and reverted.